docs/CI_next.md

```diff
@@ -0,0 +1,45 @@
+## Introduction
+
+The Continuous Integration (CI) of Confidential Containers (CoCo) since its release 0.1.0 has evolved without major design changes despite many problem being noticed along the way, some really slowing down the development/releases, but we kept applying fixes and small improvements for the sake of just supporting the growth of the community and the initial releases. With the emerging of the "merge to main" initiative and recent disruptive changes on the CI of Kata Containers on main branch (e.g. migration from Jenkins to Github Actions) our CI reached a major breakage point.
+
+On the kick-off meeting of the recently created [CoCo CI working group](https://docs.google.com/document/d/1gVuJXZzdyZcBg0Vje6n3-uL-SYdgxHzzEw9moU2ycoE) we community seemed to agreed that this is the time to re-design the CI to fix the problems that has hurt our development process. This document aims to capture the CI vision that was brought on that meeting as well as on the subsequents. On the next sections you will 
+
+## The current state of affairs
+
+But what's actually the current CoCo's CI?
+
+There are two CIs that are considered:
+
+ 1. Kata Containers CI where jenkins jobs are triggered when a pull request to kata-containers/kata-containers' CCv0 branch. Each job implement the entire workflow (from build to test) that will be described below.
+ 1. Operator CI where jenkins jobs are triggered when a pull request to confidential-containers/operator.
+
+### Kata Containers CI for CoCo
```

Comment:
One thing we discussed was also to decouple the Kata and CoCo op testing, wouldn't it make sense to create 2 documents:

* in the kata repo describing what kata CI covers and what issues should be tackled (IMO ideally via GH issues linked to that document)?
* in this repo describing what CoCo op CI covers and what issues should be covered?

In each document we should briefly mention what used to be covered, why it shouldn't be and link to the other repo for details.